release.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Script could be run from any directory.
+cd "$(dirname "$0")"
+
+# Configure the script
+properties="gradle.properties"
+changelog="CHANGELOG.md"
+readme="README.md"
+repository_url="https://github.com/RedMadRobot/gradle-version-catalogs"
+files_to_update_version=("$properties" "$readme")
+
+#region Utils
+function property {
+  grep "^${1}=" "$properties" | cut -d'=' -f2
+}
+
+function replace() {
+  # Escape linebreaks
+  local replacement=${2//$'\n'/\\\n}
+  # Portable in-place edit.
+  # See: https://stackoverflow.com/a/4247319
+  sed -i".bak" -E "s~$1~$replacement~g" "$3"
+  rm "$3.bak"
+}
+
+function diff_link() {
+  echo -n "$repository_url/compare/${1}...${2}"
+}
+#endregion
+
+# 0. Fetch remote changes
+echo "Updating local repository..."
+git fetch -p origin
+git switch main
+git pull --rebase origin
+echo "Repository updated."
+echo
+
+# 1. Calculate version values for later
+last_version=$(property "version")
```

Comment:
_Minor (close it once you've read it)_

wow, the fun `property` is really cool!